parsing/g_constr.mlg

```diff
@@ -285,39 +285,39 @@ GRAMMAR EXTEND Gram
       | id = global -> { UNamed (GType id) } ] ]
   ;
   fix_decls:
-    [ [ dcl = fix_decl -> { let (id,_,_,_,_) = dcl.CAst.v in (id,[dcl.CAst.v]) }
-      | dcl = fix_decl; "with"; dcls = LIST1 fix_decl SEP "with"; "for"; id = identref ->
+    [ [ dcl = fix_body -> { let (id,_,_,_,_) = dcl.CAst.v in (id,[dcl.CAst.v]) }
+      | dcl = fix_body; "with"; dcls = LIST1 fix_body SEP "with"; "for"; id = identref ->
         { (id,List.map (fun x -> x.CAst.v) (dcl::dcls)) } ] ]
   ;
   cofix_decls:
-    [ [ dcl = cofix_decl -> { let (id,_,_,_) = dcl.CAst.v in (id,[dcl.CAst.v]) }
-      | dcl = cofix_decl; "with"; dcls = LIST1 cofix_decl SEP "with"; "for"; id = identref ->
+    [ [ dcl = cofix_body -> { let (id,_,_,_) = dcl.CAst.v in (id,[dcl.CAst.v]) }
+      | dcl = cofix_body; "with"; dcls = LIST1 cofix_body SEP "with"; "for"; id = identref ->
         { (id,List.map (fun x -> x.CAst.v) (dcl::dcls)) } ] ]
   ;
-  fix_decl:
+  fix_body:
     [ [ id = identref; bl = binders_fixannot; ty = type_cstr; ":=";
-        c = operconstr LEVEL "200" ->
+        c = term LEVEL "200" ->
```

Comment:
Not sure I fully understand you.  Even before we began updating the doc, it used `term` everywhere rather than `operconstr`.  Renaming `c` to `body` would be clearer.  I only looked at changing nonterminal names.  If you like I can take another pass through this and adjust some of the local variable names to be more mnemonic.

Reply:
My comment is on the wrong line. I meant the change fix_decl -> fix_body, i.e. in, `id binders type := c`, it is only `c` which is commonly called the body. And since `fix_decls` is kept and is a `with`-separated list of `fix_decl`, I'm wondering why not to keep `fix_decl`.

Reply:
OK, will do.